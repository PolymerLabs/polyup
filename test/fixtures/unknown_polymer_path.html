<polymer-element name='simple-elem' noscript>
  <template>
    <div layout flex></div>
  </template>
</polymer-element>
