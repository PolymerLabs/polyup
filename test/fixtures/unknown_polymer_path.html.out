  <!--
      TODO(polyup): unable to infer path to components
      directory. This import path is probably incomplete.
   -->
  <link rel="import" href="iron-flex-layout/iron-flex-layout.html">
<dom-module id="simple-elem">
  <style>
    /* TODO(polyup): For speed, consider reworking these styles with .classes
                     and #ids rather than [attributes].
    */
    [layout] {
      @apply(--layout);
    }
    [layout][flex] {
      @apply(--layout-flex);
    }
  </style>
  <template>
    <div layout="" flex=""></div>
  </template>
  <script>
    Polymer({ is: 'simple-elem' });
  </script>
</dom-module>
